.github/workflows/holonix-update.yaml

```diff
@@ -79,6 +80,15 @@ jobs:
           git-token: ${{ secrets.HRA_GITHUB_TOKEN }}
           committer: "Holochain Release Automation <hra+gh@holochain.org>"
           title: "Update Holonix versions on ${{ inputs.branch }}"
+          body: |
```

Comment:
Some helper text for how to use the `command-listener` pipeline because I can never remember when I need it :)